products/jbrowse-web/src/Loader.tsx

```diff
@@ -137,6 +138,8 @@ export function Loader() {
       if (configText) {
         try {
           config = JSON.parse(configText)
+          const configUri = new URL(configLocation.uri, window.location.origin)
+          addRelativUris(config, configUri)
```

Comment:
Missing "e" in relative

```suggestion
          addRelativeUris(config, configUri)
```